client/my-sites/pages/virtual-page/index.tsx

```diff
@@ -112,7 +115,7 @@ const VirtualPage = ( {
 			<div className="page__main">
 				<a
 					className="page__title"
-					href={ editorUrl }
+					href={ isAdmin ? editorUrl : site.URL }
 					title={ translate( 'Edit %(title)s', {
```

Comment:
This "Edit xxx" title; does it matter if it is read only? (for non-admins)

Reply:
Good catch! I'll find if there are already `'View %(title)s'`string to reuse